### PR TITLE
output attached for Equal number occurances

### PR DESCRIPTION
**Pull request by jadhavrajashree40**
Equal number occurances

### CODE REVIEW

File: JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                        firstCount++;
+                    }
+                    secondCount++;
+                }
+            }
+            if (firstCount != secondCount) {
+                output = false;
+                return output;
+            }
+
```
Comment (rohitlamsonge66) [inline JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java:28]: remove extra space in complete Program.

File: JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        }
+        return output;
+    }
+
+    public static void main(String[] args) {
+        boolean output = EqualNumberOccurances("abacbc");
+        if (output == false) System.out.println("All Characters Have Different Number of Occurrences");
+
```
Comment (rohitlamsonge66) [inline JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java:36]: remove extra space in complete Program.

File: JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java (hunk 1 above)
Comment (jadhavrajashree40) [reply JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java:28]: done

File: JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java (hunk 2 above)
Comment (jadhavrajashree40) [reply JavaBasicProgram/src/EqualNumberOccurances/EqualOccurances.java:36]: done

**Pull request closed without merging**